sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -65,6 +65,16 @@
      */
     private final KeyVaultCertificates keyVaultCertificates;
 
+    /**
+     * Stores the Jre key store certificates.
+     */
+    private final JreCertificates jreCertificates;
+
+    /**
+     * Stores all the certificates.
+     */
+    private final List<AzureCertificates> allCertificates;
```

Comment:
Can we delete `classpathCertificates`, `keyVaultCertificates` and `jreCertificates`, 
only use `allCertificates` here? 

i.e. change the 3 fields to local variable.